front-end/.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+    "parser": "babel-eslint",
+    "extends": "react-app",
+    "env": {
+        "es6": true,
+        "browser": true,
+        "node": true
+      },
+      "rules": {
+        "no-underscore-dangle": 0,
+        "import/no-extraneous-dependencies": "off",
```

Comment:
Not neccessarily a big issue for forgotten import, but a warning will really help us be mindful of what we really need